<launch>
	<rosparam command="load"
		file="$(find i2c_sensor)/launch/TeraRangerDuo_parameter.yaml" />

	<node pkg="i2c_sensor"
		name="teraranger_sensor_node"
		type="i2c_sensor_node"
		output="screen"
	/>
</launch>
